### DISCUSSION TITLE
Disappeared after update

### DISCUSSION THREAD

**Discussion by loxley111** [SteamDeckHomebrew/decky-loader#394] [General]
Hi.

I updated to the latest version (it may have been a prelease) and now Decky Loader has disappeared. I tried to uninstall and reinstall and have restarted many times but still not working - please can someone help?  As really miss it!  Many thanks!

**Comment by josepjay** [comment]
Same issue

**Reply by loxley111** [reply]
Not just me then.  Can you post on here if you find a fix?  Thanks so much

**Reply by josepjay** [reply]
Did u fixed it ?

**Reply by KidLeader009** [reply]
any update about this issue guys?

**Reply by GrizzlyFoxCat** [reply]
Same here.

**Reply by AAGaming00** [reply]
rebooting your deck should fix it

**Reply by loxley111** [reply]
Sorry, forgot to say it was working - but now it has gone again, following a steam update - and reinstalling isent working

**Reply by KidLeader009** [reply]
same here, decky loader missing again

**Reply by Locoxella** [reply]
Wasn´t working for me neither. Maybe is not the same for the rest, but in my case the problem was combining Steam Beta version with Decky Loader Stable version. Using Steam stable and Decky stable might be your safest bet to not get any "breaking update". It is working right now if you use Steam beta combined with Decky loader pre release. Try any combination other than Steam beta with Decky loader stable to see if it gets fixed for you also (that is, before the fix gets to the stable decky loader version).

**Reply by jimmt41** [reply]
Same here, but what is interesting is while the Deckyloader menu is no longer visible the plugins are working. I noticed Autoflatpack updating 11 packages. I can right click and see options for changing artwork from Steamgriddb. My banners and theme are working. BTW Shutting down and turning back on does not resolve.

**Reply by AndrewChamp** [reply]
> Same here, but what is interesting is while the Deckyloader menu is no longer visible the plugins are working. I noticed Autoflatpack updating 11 packages. I can right click and see options for changing artwork from Steamgriddb. My banners and theme are working. BTW Shutting down and turning back on does not resolve. 

I noticed the same behavior. Any update on a fix?

**Reply by kmccarty24** [reply]
I’m also having issues with this. I’m on the beta (feb24) channel and I’ve tried reinstalling but it simply doesn’t show. 

I can also confirm that some of the plugins are still operating. Not all however

**Reply by tabby2nd** [reply]
Same here. On steam deck stable channel and today (mar 10 2024) realized my plugins menu was missing, I assume due to a recent update. It couldn't have been that long ago, I saw notifications for plug-in updates within the last month for sure. Also I recently noticed extra lag in games that used to run perfectly with my plugins tweaked a bit. Wondering if that is why. Would greatly appreciate a fix, I'll try to message steam about it too. If anyone knows the best way to do that I'd appreciate the help. Thanks for the loader, and I appreciate the work the author and everyone else involved has done!

Edit: forgot to mention what happens when I try to reinstall the loader, it basically does nothing at all when I click it to execute the file
